src/ItemsStorage.php

```diff
@@ -28,6 +28,7 @@
  *     createdAt: int|string,
  *     updatedAt: int|string
  * }
+ * @psalm-type ItemsIndexedByName = array<string, Permission|Role>
```

Comment:
Move it and types in `getChildren` and other methods to `yiisoft/rbac`

Reply:
https://github.com/yiisoft/rbac/pull/167